src/applications/personalization/profile360/components/ProfileView.jsx

```diff
@@ -110,11 +113,14 @@ class ProfileView extends React.Component {
                 hero={hero}
                 militaryInformation={militaryInformation}
               />
-              <ProfileTOC militaryInformation={militaryInformation} />
+              <ProfileTOC
+                militaryInformation={militaryInformation}
+                isDirectDepositEligible={isDirectDepositEligible}
+              />
               <div id="contact-information" />
               <ContactInformation />
               <div id="direct-deposit" />
-              <PaymentInformation />
+              {isDirectDepositEligible && <PaymentInformation />}
```

Comment:
The logic to show/hide the `PaymentInformation` component has been moved to the parent component and the ability to render null has been removed from the `PaymentInformation` component.